File: services/social-service/src/Core/Domain/Shared/Exception/FieldRequiredException.php
<?php

declare(strict_types=1);

namespace Core\Domain\Shared\Exception;

use Core\Domain\Shared\AppException\AppException;
use Core\Domain\Shared\AppException\AppExceptionStatus;

class FieldRequiredException extends AppException
{
    public function __construct(string $entity, string $field)
    {
        parent::__construct(AppExceptionStatus::INVALID_ARGUMENT, 'field.required', [$entity, $field]);
    }
}

File: services/media-service/src/main/java/dev/animedia/shared/domain/exception/FieldRequiredException.java
package dev.animedia.shared.domain.exception;

import dev.animedia.shared.domain.appexception.AppException;
import dev.animedia.shared.domain.appexception.AppExceptionStatus;

public class FieldRequiredException extends AppException {
	public FieldRequiredException(String fieldName) {
		super(AppExceptionStatus.INVALID_ARGUMENT, "field.required.exception", fieldName);
	}
}
